examples/all-clusters-app/nrfconnect/README.md

```diff
@@ -248,14 +264,15 @@ Complete the following steps to build the sample:
 
 2.  Run the following command to build the example, with _build-target_ replaced
     with the build target name of the Nordic Semiconductor's kit you own, for
-    example `nrf52840dk_nrf52840`:
+    example `nrf52840dk/nrf52840`:
 
-         $ west build -b build-target
+         $ west build -b build-target --sysbuild
```

Comment:
nit: Of course it won't hurt to have some redundancy here, but in general (except from some unusual cases :) ) the sysbuild is used by default. Maybe we could add some note about (it affects the development and related tooling less without --sysbuild flag).

Reply:
Unfortunately sysbuild is not used by default :) And sadly this is necessary to add if you are outside of nRF Connect SDK repo (--sysbuild is added only for dependent repos to NCS, and project-connectedhomeip is not one of them).